docs/glueops-platform-administrator/configuration/export-logs.md

```diff
@@ -0,0 +1,292 @@
+  parserSelector:
+    matchLabels:
+      logging.glueops.dev: 'glueops-core-primary' # needs to match the label used in ClusterParser
+  service:
+    httpServer: true
+    parsersFile: parsers.conf
+    httpPort: 45072 # see FluentBit metrics port configuration
+    # storage:
+    #  path: "/host/fluent-bit-buffer/"
+    #  backlogMemLimit: "50MB"
+    #  checksum: "off"
+    #  deleteIrrecoverableChunks: "on"
+    #  maxChunksUp: 128
+    #  metrics: "on"
+    #  sync: normal
+---
+```
+
+## Deploying the fluentbit daaemonset to ship logs:
+
+```yaml title="Define Fluentbit daemonset (this actually deploys pods and takes up compute resources in the cluster"
+apiVersion: rbac.authorization.k8s.io/v1
+apiVersion: fluentbit.fluent.io/v1alpha2
+kind: FluentBit
+metadata:
+  name: glueops-core-primary # Must be unique among all objects of same type
+  labels:
+    logging.glueops.dev: 'glueops-core-primary'  # needs to match the label used in ClusterFluentBitConfig 
+  namespace: glueops-core-fluent-operator
+spec:
+  affinity:
+    nodeAffinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+        nodeSelectorTerms:
+          - matchExpressions:
+              - key: node-role.kubernetes.io/edge
+                operator: DoesNotExist
+  dnsPolicy: ClusterFirstWithHostNet
+  fluentBitConfigName: 'glueops-core-fluent-bit-config-primary' # needs to match the ClusterFluentBitConfig object name
+  hostNetwork: true
+  metricsPort: 45072 # needs to be unique across the cluster AND match the httpPort defined in ClusterFluentBitConfig
+  envVars:
+  - name: KEY_NAME # keyvalue pairs of environment variables can be set here. Ex. AWS credentials and more.
+    value: "VALUE"
+  ports:
+    - containerPort: 45073 # needs to be unique across the cluster
+      name: "fluentbit"
+  image: "docker.io/kubesphere/fluent-bit:v2.2.0" # to make debugging easier, add a -debug to the suffix of the image name. Ex. docker.io/kubesphere/fluent-bit:v2.2.0-debug
+  positionDB:
+    hostPath:
+      path: /var/lib/fluent-bit/
+  resources:
+    limits:
+      cpu: 500m
+      memory: 200Mi
+    requests:
+      cpu: 10m
+      memory: 25Mi
+  tolerations:
+    - operator: Exists
+  # volumes:
+  #   - name: hostbuffer
+  #     hostPath:
+  #       path: /tmp/fluent-bit-buffer
+  # volumesMounts:
+  #   - mountPath: /host/fluent-bit-buffer
+  #     mountPropagation: HostToContainer
+  #     name: hostbuffer
+---
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  namespace: glueops-core-fluent-operator
+  name: glueops-core-primary # Must be unique among all objects of same type
+spec:
+  endpoints:
+    - port: metrics
+      path: /api/v2/metrics/prometheus
+      interval: 5s
+  selector:
+    matchLabels:
+      logging.glueops.dev: 'glueops-core-primary'  # needs to match the label used in ClusterFluentBitConfig 
+```
+
+
+
+Additional resources:
+- CRDs: https://doc.crds.dev/github.com/fluent/fluent-operator@v2.7.0
```

Comment:
we could tell them to check the version we're using and select that version of the docs instead of pinning

Reply:
Updated the note.